src/mjolnir/graphfilter.cc

```diff
@@ -908,6 +912,7 @@ void GraphFilter::Filter(const boost::property_tree::ptree& pt) {
 
   // TODO: thread this. Could be difficult due to sequence creates to associate nodes
 
+  SCOPED_TIMER();
```

Comment:
do we want this one? im not sure on these where we have a top level one and then one in each of the underlying methods. i guess it doesnt hurt!